src/aiori-HDF5.c

```diff
@@ -183,6 +213,34 @@ static void *HDF5_Open(char *testFileName, IOR_param_t * param)
         hid_t *fd;
         MPI_Comm comm;
         MPI_Info mpiHints = MPI_INFO_NULL;
+        char *daos_vol_dir = NULL;
+        char daos_vol_lib_path[1024] = "\0";
+        herr_t (*H5daos_set_object_class)(hid_t, char *);
+        HDF5_options_t *o;
+
+        /* Get the pathname of DAOS-VOL */
```

Comment:
i don't get what you need to do all of this.
i thought this is done by the vol plugin itself if the env variables are set. otherwise you use native vol. so why do you need to modify ior to do this?

Reply:
Please see my reply below.